api/task.go

```diff
@@ -87,12 +87,13 @@ type TaskDatabasePITRCutoverPayload struct{}
 // TaskDatabaseCreatePayload is the task payload for creating databases.
 type TaskDatabaseCreatePayload struct {
 	// The project owning the database.
-	ProjectID    int    `json:"projectId,omitempty"`
-	DatabaseName string `json:"databaseName,omitempty"`
-	Statement    string `json:"statement,omitempty"`
-	CharacterSet string `json:"character,omitempty"`
-	Collation    string `json:"collation,omitempty"`
-	Labels       string `json:"labels,omitempty"`
+	ProjectID      int    `json:"projectId,omitempty"`
+	DatabaseName   string `json:"databaseName,omitempty"`
+	CollectionName string `json:"collectionName,omitempty"`
```

Comment:
Why this is named Collection and TableName in the CreateDatabaseContext.

My feeling is this should be named as TableName to avoid being mongo specific

Reply:
Done.